commands/ListRoutes/PrettyRenderer.ts

```diff
@@ -1,28 +1,37 @@
+/*
+ * @adonisjs/core
+ *
+ * (c) AdonisJS
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
 import { BaseCommand } from '@adonisjs/ace'
-import { ApplicationContract } from '@ioc:Adonis/Core/Application'
-import { SerializedRouter } from './ListRoutes'
+import type { SerializedRoute } from './ListRoutes'
 
 export class RoutesPrettyRenderer {
-  constructor(
-    private terminalWidth: number,
-    private application: ApplicationContract,
-    private colors: BaseCommand['colors'],
-    private logger: BaseCommand['logger'],
-    private verbose: boolean
-  ) {}
+  constructor(private command: BaseCommand, private verbose: boolean, private maxWidth?: string) {}
+
+  /**
+   * Returns the terminal width
+   */
+  private getTerminalWidth() {
+    return this.maxWidth || process.stdout.columns || 80
+  }
 
   /**
    * The colors associated with each HTTP method
    */
-  private methodColors = {
-    ANY: this.colors.red.bind(this.colors),
-    GET: this.colors.blue.bind(this.colors),
-    HEAD: this.colors.white.bind(this.colors),
-    OPTIONS: this.colors.white.bind(this.colors),
-    POST: this.colors.yellow.bind(this.colors),
-    PUT: this.colors.yellow.bind(this.colors),
-    PATCH: this.colors.yellow.bind(this.colors),
-    DELETE: this.colors.red.bind(this.colors),
+  private methodColors: Record<string, keyof BaseCommand['colors']> = {
+    ANY: 'red',
+    GET: 'blue',
+    HEAD: 'white',
+    OPTIONS: 'white',
```

Comment:
What's the output of those colors on a white color scheme?

Reply:
Yeah. I have kept color and UI changes for another PR